proxy/src/coco/control.rs

```diff
@@ -71,7 +71,7 @@ pub fn setup_fixtures(
 /// the coco project.
 pub fn replicate_platinum(
     peer: &PeerApi,
-    key: keys::SecretKey,
+    key: &keys::SecretKey,
```

Comment:
I try and keep `SecretKey` as owned because we don't want to keep in memory.

Reply:
Is that a security measure, if so, does it make sense since these functions should only be run by tests?

Reply:
Ya, fair :+1: